golang/load-balancer/docker-compose.yaml

```diff
@@ -0,0 +1,86 @@
+services:
```

Comment:
One thing I don't like here is that we have 3 identical docker-compose.yaml files (1 per language example), in the restructure ticket (https://aerospike.atlassian.net/browse/GRAPH-1592) the idea was to have a root directory called "examples" and then dir per language, then we could have another directory called infra/common/setup/etc... with common files for multiple examples, in that way if we introduce a change (e.g. bump aerospike version) we won't have to do it in multiple files (and potentially break alignment).

Not sure if at this point we want to introduce "examples" root folder (and update docs as well) or just create that infra/common/... folder and put the common files there. For sure we shouldn't duplicate files.